apis_core/apis_entities/abc.py

```diff
@@ -21,6 +24,24 @@ class E21_Person(LabelBaseModel, models.Model):
     class Meta:
         abstract = True
 
+    @property
+    def node(self):
+        if self.uri_set.all():
+            return URIRef(self.uri_set.first().uri)
```

Comment:
If we stick with the serialization per attribute and keep the definition of the RDF serialization with the models - which has the disadvantage that we can have only one RDF serialization - we need to return here one canonical URI (aka the one that contains the base domain of the project), because other entities - such as the triples - need to know how to construct the correct URI from an ID.